### ISSUE TITLE
Help please, when running Neko locally, it times out at login

### ISSUE THREAD

**Issue by SpiderSuave** [m1k1o/neko#431]
It's been running great via remote VPS, but I'd like to run this locally and I can't seem to get it to work. Any help please is much appreciated.

I'm using the same Neko Rooms installation script that I've always used on a VPS. The installation finishes fine and I'm entering my lan address as the domain name. From my browser when I go to that lan address, I get the Neko Rooms management page and I can launch the room fine, but then at the login prompt it times out and I can't get into the room.

I've also installed it on a VirtualBox machine using that machine's lan address and the same thing happens at the login.

Thanks again for any help.

**Comment by m1k1o** [comment]
Neko rooms installation sripr assumes that you have public IP and uses that. To use your own local you need to ass `NEKO_ROMS_NAT1TO=<local-ip>` so that your rooms can correctly let the clients know their local IP.

**Comment by wiktormalyska** [comment]
what if i have same issue but i reverse proxy it? should i set env with ip of this server?

**Comment by SpiderSuave** [comment]
> Neko rooms installation sripr assumes that you have public IP and uses that. To use your own local you need to ass `NEKO_ROMS_NAT1TO=<local-ip>` so that your rooms can correctly let the clients know their local IP.

Hi, thank you. That worked great and was working fine until just a few days ago. For some reason now my LAN address in my browser address bar doesn't even start Neko-Rooms. Please, can you suggest how I can fix it?

It was working fine by just adding this line in neko-rooms.sh under the NEKO_IMAGES section where the echo statements are: 
 echo "NEKO_ROOMS_NAT1TO1=###.###.#.###" with the # being my LAN IP address.

Thanks again for your help.

**Comment by m1k1o** [comment]
Hello, it looks like something changed on your network. Please verify that neko rooms is running, that the IPs did not change (if you have dynamic allocation that can happen sometimes).

**Comment by SpiderSuave** [comment]
Hi. Thank you. My LAN IP address is the same. I'm on Kubuntu and ip addr shows the same address, which is what I have in neko-rooms.sh. I've also tried with 127.0.0.1 and localhost but no luck. I even did a fresh install of Kubuntu in case something else was the problem but the same thing still happens.

Could the latest update to Neko-Rooms, "fix ip for dev.", have something to do with it?

Thanks again for your help.

**Comment by m1k1o** [comment]
The latest fix has nothing to do unless you run any of the scripts inside /dev/ folder.

I assume you installed it either using install script or you just use docker compose. Do you use traefik or no traefik? But if you can't even access the neko rooms GUI, then its probably some issue on the network. Can you access it directly on the machine with `curl http://127.0.0.1:8080` or the port that you use?

**Comment by SpiderSuave** [comment]
I've installed it using the neko-rooms.sh script. At the end of installation it says "Container nekotest-traefik-1     Started"

With curl http://127.0.0.1:8080 or :80 or with lan ip address:8080 or :80 I get nothing.

It's strange because it was working beautifully day after day until just the other day by simply putting lan ip address in the browser, then all of a sudden it wouldn't come up at all using that exact same lan ip address.

Thank you.

**Comment by SpiderSuave** [comment]
Hello again. It's apparently a problem with Apache2. When I restart Apache2, port 80 opens, but then when I install Neko-Rooms via the neko-rooms.sh script, I get a binding error on port 80 and nothing works. The installation only succeeds if Apache2 is stopped, but then I can't bring up the app in the browser.

I appreciate any help. Thank you.

**Comment by m1k1o** [comment]
You are trying to run both on port 80. Either change port on apache to eg 81 or on neko rooms, and then you can use proxy pass for neko rooms on apache. But you need to handle https by yourself (eg using certbot) if you change traefik from port 80 to a different one.

**Comment by SpiderSuave** [comment]
> You are trying to run both on port 80. Either change port on apache to eg 81 or on neko rooms, and then you can use proxy pass for neko rooms on apache. But you need to handle https by yourself (eg using certbot) if you change traefik from port 80 to a different one.

Hi. I tried that but still no luck.

I only want to run Neko-Rooms locally which until a few days ago was working fine just by just installing it with the neko-rooms.sh script.

Installing Apache was just something I did now to see if that would help fix the problem of Neko-Rooms not starting.

I appreciate your help. I just keep trying and trying here and I can't figure out what's going on.

**Comment by SpiderSuave** [comment]
Okay I just booted into KDE Neon live and the app works fine again, so it was definitely something changed on my side but I couldn't figure out what it was.

I appreciate your help though and thanks again.

**Comment by SpiderSuave** [comment]
Okay, it all got fixed by just deleting everything in the directory and re-downloading the script. The install worked fine and also neko-rooms launched perfectly fine from the browser.

Silly me I should've done this first but I think I thought I had already done it so I went off trying other things.

So it was apparently some file in the directory that got corrupt and all is good now.

Thank you so much for your help as always.
